src/Database/PostgreSQL/Simple/ToRow.hs

```diff
@@ -21,6 +21,8 @@ module Database.PostgreSQL.Simple.ToRow
       ToRow(..)
     ) where
 
+import Data.Foldable (toList)
```

Comment:
I'd prefer `import qualified Data.List.NonEmpty as NE` and `NE.toList`. `toList` become a method of `Foldable` quite recently, so it doesn't use efficient impl on older GHCs.